hvac/api/system_backend/init.py

```diff
@@ -96,18 +126,18 @@ def initialize(
         if recovery_shares is not None:
             params["recovery_shares"] = recovery_shares
 
-        if recovery_threshold is not None:
-            if recovery_threshold > recovery_shares:
-                error_msg = "value for recovery_threshold argument be less than or equal to recovery_shares argument"
-                raise ParamValidationError(error_msg)
-            params["recovery_threshold"] = recovery_threshold
-
-        if recovery_pgp_keys is not None:
-            if len(recovery_pgp_keys) != recovery_shares:
-                raise ParamValidationError(
-                    "length of recovery_pgp_keys list argument must equal recovery_shares value"
-                )
-            params["recovery_pgp_keys"] = recovery_pgp_keys
+            if recovery_threshold is not None:
+                if recovery_threshold > recovery_shares:
+                    error_msg = "value for recovery_threshold argument must be less than or equal to recovery_shares argument"
+                    raise ParamValidationError(error_msg)
```

Comment:
Why use an explicit error_msg variable here when the current file has previously not done so? Its not a breaking instance of code just stylistic.

Reply:
looking at left side in the side-by-side diff, it seems it's because the previous code did in fact use this variable, and I just changed the message minimally but didn't remove use of the var. I don't feel strongly about it, if you want to put in a suggested change in the comment that removes the variable I'll probably accept it.